### PR TITLE
Ui setup updates

### PR DESCRIPTION
**Pull request by mzedel**
this should now allow running the service with just `make docker-dev` (maybe remove the `image: tetra-*` defintions to allow automatic builds on a `docker-compose up`-like command), with a hot reloading (on filechange on the host file system) site served on http://localhost:7375

to fully interact with the api this: https://github.com/lluiscampos/tetra/pull/3 also needs to be merged, since the api runs under localhost:7374 - so a completely different origin ;)

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -10,20 +10,36 @@
     "react-router": "^2.4.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.9.0",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "webpack": "^1.13.1"
+        "@babel/core": "^7.6.4",
```
Comment (lluiscampos) [inline ui/package.json:13]: Not that it matters a lot, but the tab size is inconsistent.

File: ui/package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-    "react-router": "^2.4.1"
-  },
-  "devDependencies": {
+    "name": "tetra-ui",
+    "version": "1.0.0",
+    "description": "the dashboard for tetra",
+    "main": "app/main.js",
+    "dependencies": {
+        "canon-react": "^1.0.0",
+        "react": "~16.8.6",
```
Comment (lluiscampos) [inline ui/package.json:8]: That was quite an update: from 0.14 to 16.8...

**Review by lluiscampos — APPROVED** [review]
Thank you very much for your contribution.

Even from my little javascript expertise, I can tell the end result looks cleaner :+1:

File: ui/package.json (hunk 1 above)
Comment (mzedel) [reply ui/package.json:13]: thanks for catching that - I'll add some formatting help in a follow up PR :)

**Pull request merged**